lib/oban/telemetry.ex

```diff
@@ -305,16 +305,20 @@ defmodule Oban.Telemetry do
   Attach a logger at the `:debug` level:
 
       :ok = Oban.Telemetry.attach_default_logger(:debug)
+
+  Attach a logger with a specific oban prefix:
+
+      :ok = Oban.Telemetry.attach_default_logger(:info, [:my_oban])
   """
   @doc since: "0.4.0"
   @spec attach_default_logger(Logger.level()) :: :ok | {:error, :already_exists}
-  def attach_default_logger(level \\ :info) do
+  def attach_default_logger(level \\ :info, telemetry_prefix \\ [:oban]) do
```

Comment:
Using two defaults makes it impossible to set the prefix without setting the level. What if we turn this into a keyword list instead?

```elixir
  def attach_default_logger(level) when is_atom(level) do
    attach_default_logger(level: level)
  end

  def attach_default_logger(opts) when is_list(opts) do
    # ...
```

Reply:
That's a good idea! I was afraid of breaking compatibility, but your default solves it :)